ydb/docs/en/core/yql/reference/yql-core/syntax/_includes/create_table.md

```diff
@@ -159,6 +159,28 @@ WITH (
 );
 ```
 
+Creating a table with enabled [TTL](../../../../concepts/ttl.md) for automatically removing outdated records from the table. A record is deleted when the value of one of the columns (configurable by the user) of type: `Date`, `Datetime`, or `Timestamp` becomes greater than the current time (Epoch time). The lifespan of records can be extended (globally for the entire table) by the value specified in the `Interval` function:
+```sql
+CREATE TABLE my_table (
+    id Uint64,
+    title Utf8,
+    expire_at Timestamp
+    PRIMARY KEY (id)
+)
+WITH (
+    TTL = Interval("PT0S") ON expire_at
```

Comment:
This is a corner case example. Let's add a more typical one first. For example, there's a column with an event time (timestamp at the moment of insert), and we want to discard events older than one week.